README.md

```diff
@@ -59,6 +61,13 @@ _Default: no timeout (other timeouts will eventually close the connection)._
 Sets timeout (in seconds) for establishing TCP connection to target website. Affects all requests.  
 _Default: 20 seconds._
 
+- **upstream [https://username:password@upstreamproxy.site:443]**  
+Sets upstream proxy to route all requests through it. For probe resistance reasons,
```

Comment:
Would "Sets upstream proxy to route all requests through it." -> "Sets upstream proxy to route all **forwardproxy** requests through it" be good enough clarification?

> It might be worth checking whether there is a way to (optionally?) apply this dialer to all HTTP requests generated by Caddy.

Not sure how if we want to have such a Caddy-wide feature in `forwardproxy`, especially since changed behavior might give out the fact that there is a forwardproxy.

Reply:
Yes, that's fine.  I agree that having a Caddy-wide behavior change in `forwardproxy` would be strange, but it might be a good idea even if this isn't the right place for it.